docs/README.md

```diff
@@ -37,110 +37,31 @@ which are defined in [website/layouts/index.redirects](./webiste/layouts/index.r
 and are build into a `_redirects` file when the Hugo build happens via
 `make production-build` or `make preview-build`.
 
-### How to Edit and Test
-
-Because of the different components at play there are a few different ways to
-test/view the website. The choice depends largely on what is being modified:
-
-#### Full Site Preview
-
-Go to [Netlify](https://www.netlify.com/) and log-in. Link to your public fork of
-OPA on github and have it deploy a site. As long as it is public this is free
-and can be configured to deploy test branches before opening PR's on the official
-OPA github repo.
-
-This approach gives the best simulation for what the website will behave like once
-code has merged.
+## Site updates
 
+The OPA site is automatically published using [Netlify](https://netlify.com). Whenever
+changes in this directory are pushed to `master`, the site will be re-built and
+re-deployed.
 
-#### Modifying `content` (*.md)
+## How to Edit and Test
+### Preview Markdown `content` (*.md)
 
-The majorify of this can be done with any markdown renderer (typically built-in or
-a plug-in for IDE's). The rendered output will be very similar to what Hugo will
+The majorify of this can be done with any markdown renderer (typically built into or
```

Comment:
s/majorify/majority/ -- though I kind of like "majorify"

Reply:
Fixed 😆